BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -3,6 +3,7 @@
 #include "BSP_CAN.h"
 #include "stm32f4xx.h"
 #include "os.h"
+#include "BSP_Lights.h"
```

Comment:
Should this be removed if it was only used for testing purposes?

Reply:
yes, good catch